addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/__tests__/environment-sc-cidr-service.test.js

```diff
@@ -869,4 +869,49 @@ describe('EnvironmentScCidrService', () => {
       expect(response).toEqual(responseObj);
     });
   });
+  describe('authorizeIngressRuleWithSecurityGroup', () => {
+    it('should call authorize security group', async () => {
+      service.authorizeSecurityGroupIngress = jest.fn();
+      service.getEc2Client = jest.fn(() => {
+        return {};
+      });
+      await service.authorizeIngressRuleWithSecurityGroup({}, 'envId', {}, 'groupId');
+      expect(service.authorizeSecurityGroupIngress).toHaveBeenCalled();
```

Comment:
Verify that `authorizeSecurityGroupIngress` was called with correct arguments.

Reply:
Incorporated the suggestion and updated the PR